src/main/java/com/homework/springboot/service/impl/EmailServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.homework.springboot.service.impl;
+import com.homework.springboot.service.EmailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Service
+@RequiredArgsConstructor
+public class EmailServiceImpl implements EmailService {
+
+    private final JavaMailSender emailSender;
+
+    public EmailServiceImpl() {
+        emailSender = null;
+    }
+
+    @Override
+    public void send(String to, String subject, String text) {
+        MimeMessage message = this.emailSender.createMimeMessage();
+        MimeMessageHelper helper = new MimeMessageHelper(message);
+        try {
+            helper.setTo(to);
+            helper.setSubject(subject);
+            helper.setText(text);
+            this.emailSender.send(message);
+        } catch (MessagingException messageException) {
+            throw new RuntimeException(messageException);
```

Comment:
It is better to create your own exception and use it here.
For example MessagingRuntimeException